config/configs/coherence.exs

```diff
@@ -2,7 +2,7 @@ use Mix.Config
 
 # %% Coherence Configuration %%   Don't remove this line
 config :coherence,
-  user_schema: CodeSponsor.Coherence.User,
+  user_schema: CodeSponsor.Schema.User,
```

Comment:
For sure, should be tested. Running it locally it seemed like auth still worked but could be that another part doesn't. (User was the first one I changed since it seemed easy to manually test locally)

Reply:
I tested it and it appears to not like the change

```[error] GenServer Coherence.RememberableServer terminating
** (UndefinedFunctionError) function CodeSponsor.Coherence.Rememberable.hash/1 is undefined (module CodeSponsor.Coherence.Rememberable is not available)
    CodeSponsor.Coherence.Rememberable.hash("2g6Qbl3Jbi")
    (coherence) lib/coherence/controllers/session_controller.ex:326: Coherence.SessionController.validate_login/3
    (coherence) lib/coherence/controllers/session_controller.ex:230: Coherence.SessionController.do_rememberable_callback/5
    (coherence) lib/coherence/rememberable_server.ex:27: Coherence.RememberableServer.handle_call/3
    (stdlib) gen_server.erl:636: :gen_server.try_handle_call/4
    (stdlib) gen_server.erl:665: :gen_server.handle_msg/6
    (stdlib) proc_lib.erl:247: :proc_lib.init_p_do_apply/3
Last message (from #PID<0.1116.0>): {:callback, #Function<4.105879294/0 in Coherence.SessionController.rememberable_callback/5>}
State: nil
Client #PID<0.1116.0> is alive
    (stdlib) gen.erl:169: :gen.do_call/4
    (elixir) lib/gen_server.ex:827: GenServer.call/3
    (coherence) lib/coherence/plugs/authorization/session.ex:207: Coherence.Authentication.Session.call/2
    (code_sponsor) lib/code_sponsor_web/router.ex:5: CodeSponsorWeb.Router.browser/2
    (code_sponsor) lib/code_sponsor_web/router.ex:1: anonymous fn/1 in CodeSponsorWeb.Router.__match_route__/4
    (phoenix) lib/phoenix/router.ex:273: Phoenix.Router.__call__/1
    (code_sponsor) lib/code_sponsor_web/endpoint.ex:1: CodeSponsorWeb.Endpoint.plug_builder_call/2
    (code_sponsor) lib/plug/debugger.ex:99: CodeSponsorWeb.Endpoint."call (overridable 3)"/2
    (code_sponsor) lib/code_sponsor_web/endpoint.ex:1: CodeSponsorWeb.Endpoint.call/2
    (plug) lib/plug/adapters/cowboy/handler.ex:15: Plug.Adapters.Cowboy.Handler.upgrade/4
    (cowboy) /Users/eberry/Code/Gitcoin/code_sponsor/deps/cowboy/src/cowboy_protocol.erl:442: :cowboy_protocol.execute/4```

Reply:
I'll take a look at that first.

Reply:
I'll take a look at that first.

Reply:
No easy way to configure it, could metaprogram something to make it work in the same namespace as the rest of the schemas but truly, now that I get what this is doing better, it makes sense just to move it back under `Coherence` since its effectively a private object for the auth anyway.